packages/core-snapshots/src/plugin.ts

```diff
@@ -14,6 +14,7 @@ export const plugin: Container.PluginDescriptor = {
     async register(container: Container.IContainer, options) {
         const manager = new SnapshotManager(options);
 
-        return manager.make(container.resolvePlugin<PostgresConnection>("database"));
+        const connection = container.resolvePlugin<Database.IDatabaseService>("database").connection as any;
```

Comment:
This causes some resolution issues. Try `yarn dump:testnet --config ../core/src/config/testnet` through the CLI to reproduce it.

```
TypeError: Cannot read property 'connection' of null
    at Object.register (./packages/core-snapshots/dist/plugin.js:15:63)
```

Reply:
@faustbrian I've solved the issue.